lib/common_test/src/ct_util.erl

```diff
@@ -77,12 +77,13 @@
 
 -define(default_verbosity, [{default,?MAX_VERBOSITY},
 			    {'$unspecified',?MAX_VERBOSITY}]).
+-define(default_custom_stylesheet, undefined).
```

Comment:
could that default be push to ct_logs as well? could we gain something and reduce number of modules affected with this PR?